ros/processor/inventory_events_consumer.py

```diff
@@ -147,6 +166,9 @@ def process_system_details(self, msg):
                         f"{self.prefix} - Refreshed system {system.inventory_id} ({system.id}) "
                         f"belonging to account: {account.account} ({account.id}) and org_id: {account.org_id}"
                     )
+                    self.expired_del_cache_if_exists(
+                        host['id'], account.org_id
+                    )
```

Comment:
@kgaikwad Food for thought, apart from above do we need any additional code as cache-invalidation policy?

Reply:
In my opinion, it is sufficient as Expiry is defined on caching data. Do you have any corner scenario that needs to be handle separately?